eng/install-native-dependencies.sh

```diff
@@ -10,14 +10,9 @@ if [ "$1" = "Linux" ]; then
         exit 1;
     fi
 elif [ "$1" = "OSX" ]; then
-    brew update
-    if [ "$?" != "0" ]; then
-        exit 1;
-    fi
-    brew install icu4c openssl
-    if [ "$?" != "0" ]; then
-        exit 1;
-    fi
+    engdir=$(dirname "${BASH_SOURCE[0]}")
```

Comment:
the shebang at the top of this file should be `#!/usr/bin/env bash` to make sure `${BASH_SOURCE[0]}` works (it is that way in dotnet/runtime master).